src/bundler.ts

```diff
@@ -30,11 +29,6 @@ export class Bundler {
     return writeFileAsync(metadataFile, JSON.stringify(metadata));
   }
 
-  private createTarFromSrcFolderAndAddToDist(srcPath: string, distPath: string) {
-    const files = fs.readdirSync(srcPath);
-    return tar.c({ gzip: true, cwd: srcPath, file: join(distPath, 'src.tgz') }, files);
```

Comment:
If I understand this right, `src.tgz` is not used anyway for the store or when uploading a report, right? Or is it not even uploaded?

Reply:
Yes, I talked to @psurrey, @blandsbergleanix and @nikolasrist, and nobody is aware of any usage of it. It's also not used in Pathfinder and I double-checked that reports can be uploaded without any issue without a `src.tgz` in the bundle.